Dockerfile.backend

```
@@ -1,28 +1,45 @@
+WORKDIR /app
+ENV NODE_ENV=production
+
+COPY package.backend.json ./package.json
+COPY .sequelizerc .sequelizerc
+
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/server ./server
+COPY --from=builder /app/public ./public
 
 # Expose backend port
 EXPOSE ${BACKEND_PORT:-8080}
```

Hadolint (2.14.0)

[error] unexpected '}'
expecting '/', a new line followed by the next instruction, or digit

(DL1000)